packages/@ionic/cli/src/commands/package/build.ts

```diff
@@ -74,6 +77,12 @@ Customizing the build:
 Deploying the build to an App Store:
 - The ${input('--destination')} option can be used to deliver the app created by the build to the configured App Store. \
 This can be used only together with build type ${input('release')} for Android and build types ${input('app-store')} or ${input('enterprise')} for iOS.
+
+Downloading build artifacts:
+- By default once the build is complete, all artifacts are downloaded for the selected platform. ${input('aab')} and ${input('apk')} for Android \
+${input('ipa')} and ${input('dsym')} for iOS.
+- The ${input('--artifact-type')} option can be used to limit artifact downloads to only of that type. For instance, with Android, you can specify ${input('aab')} \
```

Comment:
> limit artifact downloads to only of that type
is this correct?

Reply:
Yes, if you specify artifact type it will only download that type